### PR TITLE
Correct typo for replaceProcessGroup

### PR DESCRIPTION
**Pull request by xumengpanda**
# Description

ReplaceProcessGroup procedure has typo in its help message and comment.
This PR fixex the typos.

## Type of change

- Bug fix (non-breaking change which fixes an issue)
No functional change.

## Testing
No testing since it's not changing functionality.

### CODE REVIEW

File: kubectl-fdb/cmd/root.go [hunk 1]
```diff
@@ -23,12 +23,13 @@ package cmd
...
 	"bufio"
 	"fmt"
-	"github.com/fatih/color"
 	"log"
 	"math/rand"
 	"os"
 	"time"
 
+	"github.com/fatih/color"
+
```
Comment (johscheuer) [inline kubectl-fdb/cmd/root.go:32]: Any reason to change the order of the imports?

**Review by johscheuer — CHANGES_REQUESTED** [review]
Can you please run `make clean all` to ensure everything is generated properly. You also have to change the code in v1beta2, v1beta1 is read-only.

File: kubectl-fdb/cmd/root.go (hunk 1 above)
Comment (xumengpanda) [reply kubectl-fdb/cmd/root.go:32]: nope. my VSCode seems changing this automatically.

**Comment by xumengpanda** [comment]
The PR test failure is not relevant to this PR. It seems indicating the buggify feature has bug?

 ```
[FAILED] [56.158 seconds]
Operator
/codebuild/output/src594396191/src/github.com/FoundationDB/fdb-kubernetes-operator/fdb-kubernetes-tests/tests/test_operator/operator_test.go:117
  using the buggify option to ignore a process during the restart
  /codebuild/output/src594396191/src/github.com/FoundationDB/fdb-kubernetes-operator/fdb-kubernetes-tests/tests/test_operator/operator_test.go:869
    [It] should not restart the process on the ignore list
    /codebuild/output/src594396191/src/github.com/FoundationDB/fdb-kubernetes-operator/fdb-kubernetes-tests/tests/test_operator/operator_test.go:910

  Failed after 15.036s.
  Expected
      <bool>: false
  to be true
  In [It] at: /codebuild/output/src594396191/src/github.com/FoundationDB/fdb-kubernetes-operator/fdb-kubernetes-tests/tests/test_operator/operator_test.go:942
```

**Comment by johscheuer** [comment]
> The PR test failure is not relevant to this PR. It seems indicating the buggify feature has bug?
> 
> ```
> [FAILED] [56.158 seconds]
> Operator
> /codebuild/output/src594396191/src/github.com/FoundationDB/fdb-kubernetes-operator/fdb-kubernetes-tests/tests/test_operator/operator_test.go:117
>  using the buggify option to ignore a process during the restart
>  /codebuild/output/src594396191/src/github.com/FoundationDB/fdb-kubernetes-operator/fdb-kubernetes-tests/tests/test_operator/operator_test.go:869
>    [It] should not restart the process on the ignore list
>    /codebuild/output/src594396191/src/github.com/FoundationDB/fdb-kubernetes-operator/fdb-kubernetes-tests/tests/test_operator/operator_test.go:910
> 
>  Failed after 15.036s.
>  Expected
>      <bool>: false
>  to be true
>  In [It] at: /codebuild/output/src594396191/src/github.com/FoundationDB/fdb-kubernetes-operator/fdb-kubernetes-tests/tests/test_operator/operator_test.go:942
> ```

This could also be an issue with the test setup, we already have an issue to investigate this failure.

**Pull request merged**